### ISSUE TITLE
Issue with Vite arising in Avo integration directions

### ISSUE THREAD

**Issue by brandondrew** [maglevhq/maglev-core#53]
Following the directions at https://docs.maglev.dev/integrations/avo, I ran into a an error when I got to `bin/rails g maglev:install`

```
/Users/brandon/.rbenv/versions/3.2.0/lib/ruby/gems/3.2.0/gems/vite_ruby-3.3.2/lib/vite_ruby/compatibility_check.rb:24:in `raise_unless_satisfied': vite-plugin-ruby@^3.1.2 might not be compatible with vite_ruby-3.3.2 (ArgumentError)

You may disable this check if needed: https://vite-ruby.netlify.app/config/#skipcompatibilitycheck

You may upgrade both by running:

    bundle exec vite upgrade


        raise ArgumentError, <<~ERROR
              ^^^^^^^^^^^^^^^^^^^^^^^
```

Running `bundle exec vite upgrade` did not solve anything.  Even using the environment variable suggested on the Rails Vite page didn't resolve anything:

```
VITE_RUBY_SKIP_COMPATIBILITY_CHECK=1 bin/rails s
```

`bin/vite upgrade` (also suggested by Vite-Ruby's documentation) didn't do anything because `vite` doesn't exist in `bin`. The error remained, and appeared any time I tried to run a command:

```
ArgumentError: vite-plugin-ruby@^3.1.2 might not be compatible with vite_ruby-3.3.2
```

I've never used Vite before, so I have no insight into what may be going wrong, and I'm not able to find any clues by Googling.

**Comment by brandondrew** [comment]
I tried specifying the `vite_ruby` gem to be 3.1.2, and later the `vite_rails` gem to be 3.1.2, hoping that might correspond to `vite-plugin-ruby@3.1.2`, but it didn't help.

**Comment by brandondrew** [comment]
The same thing happens immediately with this

```
rails new my-awesome-site \
  -m https://raw.githubusercontent.com/maglevhq/maglev-core/master/template.rb \
  --database=postgresql \
  --skip-action-cable
```

**Comment by brandondrew** [comment]
Or simply 

```
rails new my-awesome-site \
  -m https://raw.githubusercontent.com/maglevhq/maglev-core/master/template.rb
```

**Comment by BeatyThomas** [comment]
getting this same error trying to do a new install with this:

`rails new my-awesome-site \
  -m https://raw.githubusercontent.com/maglevhq/maglev-core/master/template.rb \
  --database=postgresql \
  --skip-action-cable`

**Comment by jamgar** [comment]
Get the same using both the Short and Long commands listed in the instructions.

**Comment by brandondrew** [comment]
@did Do you have any insight into the problem we're having?

**Comment by did** [comment]
hey all 👋 I'll take a look at it this evening.

**Comment by did** [comment]
👋 @brandondrew. 

I did 2 things:
- I released a new version of MaglevCMS gem (1.2.1) which fix a couple of issues around vite (especially, the `assets:precompile` rake task)
- I updated the `template.rb` so that it installs the latest versions of both Maglev and Puma.

I haven't had the time to re-play the Avo guide. I'll do it later today.

**Comment by did** [comment]
I'm closing this one. Don't hesitate to re-open it if the exact same issue arises again. Thanks!
